### PR TITLE
Changelog for v2.1.0.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

**Review by pyricau — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 
+Version 2.1.0
+-------------
+
+_2020-10-14_
+
+* Introduce support for rendering across subcompositions. (#104)
+* Fix for bug where layout IDs to skip were queried incorrectly. (#94 – thanks @samruston!)
+* Fix grammatical errors in documentation. (#92 – thanks @androiddevnotesfork!)
+* Upgrade Compose version being compiled and tested against to alpha05. (#106)
```
Comment (pyricau) [inline CHANGELOG.md:12]: > against to alpha05

=> against alpha05?

Comment (zach-klippenstein) [reply CHANGELOG.md:12]: "Upgrade Compose version (being compiled and tested against) to alpha05."

Comment (pyricau) [reply CHANGELOG.md:12]: makes sense now but definitely not obvious

Comment (zach-klippenstein) [reply CHANGELOG.md]: Removed the extra wording, just saying "Updated compose to x.y.z" instead, like previous changelogs.

**Comment by zach-klippenstein** [comment]
This is just a markdown change, merging before CI finishes.

**Pull request merged**